content/docs/command-reference/remote/modify.md

```diff
@@ -574,6 +574,10 @@ more information.
 
 ### Click for HDFS
 
+💡 Using a HDFS cluster as remote storage is also supported via the WebHDFS API.
+Read more about by expanding the WebHDFS section in
+[`dvc remote add`](/doc/command-reference/remote/add#supported-storage-types).
+
```

Comment:
Prob not needed in this doc. since you come here after already decided which one to use.